starter/js/atm.js

```diff
@@ -1,43 +1,76 @@
 //Begin with the document ready function
+$(document).ready(function() {
 
-    //Checking account deposit function
 
-      //On click of the depositChecking button
+  var checkingBalance = $('#checkingBalance');
```

Comment:
another good idea for naming variables is to have $ in the name for anything that is holding a jQuery object (eg `$checkingBalance = $('#checkingBalance')`